### PR TITLE
Logger and locale added

### PR DESCRIPTION
**Pull request by Devas**
Locale and logger fully finished

### CODE REVIEW

File: src/main/java/com/fourmusketeers/client/locale/LocaleManager.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+public enum LocaleManager {
+    INSTANCE;
+
+    private String resourcesPath;
+    private String localizationFileName;
+    private Properties localizationProperties;
+
+    public void setProperLocale(Locales locale ) {
+        this.resourcesPath = "src/main/resources/";
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/locale/LocaleManager.java]: System.getProperty?

File: src/main/java/com/fourmusketeers/client/locale/LocaleManager.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        this.localizationFileName = "localization" + locale.toString() + ".properties";
+        this.localizationProperties = new Properties();
+        loadLocalization();
+    }
+
+    void loadLocalization() {
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/locale/LocaleManager.java]: ResourceBundle.getBundle!!!

File: src/main/java/com/fourmusketeers/client/locale/LocaleManager.java (hunk 1 above)
Comment (Dzionek95) [inline src/main/java/com/fourmusketeers/client/locale/LocaleManager.java]: 👎

File: src/main/java/com/fourmusketeers/client/locale/LocaleManager.java (hunk 2 above)
Comment (Dzionek95) [inline src/main/java/com/fourmusketeers/client/locale/LocaleManager.java]: 👎

**Pull request merged**